### PR TITLE
DPC-1475: As a user, I want to search through provider orgs

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1475](https://jira.cms.gov/browse/DPC-1475)

To make provider organizations more manageable, users should be able to search through client organizations using name, org_id, or NPI.

### Proposed Changes

- User can filter by name, NPI, or uuid

### Change Details

- Moved provider org list into partial
- Changed port to the 5434 to our agreed `impl` port to work with API.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

filterorgs

### CODE REVIEW

**Review by dhgreene — DISMISSED** [review]
👍

File: dpc-impl/db/structure.sql [hunk 1]
```diff
@@ -11,6 +11,48 @@ SET row_security = off;
...
+    email character varying DEFAULT ''::character varying,
+    encrypted_password character varying DEFAULT ''::character varying NOT NULL,
+    sign_in_count integer DEFAULT 0 NOT NULL,
+    current_sign_in_at timestamp without time zone,
+    last_sign_in_at timestamp without time zone,
+    current_sign_in_ip character varying,
+    last_sign_in_ip character varying,
+    created_at timestamp(6) without time zone NOT NULL,
+    updated_at timestamp(6) without time zone NOT NULL
+);
```
Comment (dhgreene) [inline dpc-impl/db/structure.sql:34]: Duplicate of lines 60-75

Comment (Sun-Mountain) [reply dpc-impl/db/structure.sql:34]: Weird, locally mine doesn't have a duplicate.

**Review by dhgreene — APPROVED** [review]
⚡

**Pull request merged**